cypress/support/pages/cart_page.ts

```diff
@@ -0,0 +1,99 @@
+class CartPage {
```

Comment:
Please follow the style in https://github.com/abdullah-aladham/automationStore-CypressTS/blob/master/cypress%2Fsupport%2Fpages%2Flogin_page.ts

Define elements 

Define methods

Keep class clean and clear responsibilities